.github/workflows/ci.yaml

```
@@ -77,6 +77,11 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: Remove-Item ./target/debug/xtask.exe
 
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: editors
```

```suggestion
          name: executables
```

Also I think you should omit all sub dirs.

---

Good spot